Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ however, it has to be formatted properly to pass verification tests.
 #### Actions
 
 * Fixed `InputAction.GetTimeoutCompletionPercentage` jumping to 100% completion early ([case 1377009](https://issuetracker.unity3d.com/issues/gettimeoutcompletionpercentage-returns-1-after-0-dot-1s-when-hold-action-was-started-even-though-it-is-not-performed-yet)).
+* Fixed `IndexOutOfRangeException` when having multiple interactions on an action and/or binding in an action map other than the first of an asset ([case 1392559](https://issuetracker.unity3d.com/issues/map-index-on-trigger-and-indexoutofrangeexception-are-thrown-when-using-interaction-on-both-binding-and-its-parent-action)).
+  - Fix contributed by [Russell Quinn](https://github.com/russellquinn) in [#1483](https://github.com/Unity-Technologies/InputSystem/pull/1483).
```

Comment:
Nitpic: Should use asterisk instead of dash for list here and avoid indent or intentional sub-bullet? Why not have it as part of previous fix bullet?

Reply:
Indentation was intentional. Think details for a fix read nicer that way, in general.

Reply:
ok, just reacted to previous way of doing it, I think we should pick one way and stick to it though

Reply:
Hi, just checking if there was an anticipated timeline for this fix making it into the codebase. Thanks, Russell.

Reply:
It'll be in 1.4. We're expecting to clear out the merge queue this week to prep for that release and for it to start making its way through the release pipes end of this week or beginning of next.

Reply:
Thanks for the update!